exo/main.py

```diff
@@ -59,9 +62,21 @@
 inference_engine = get_inference_engine(inference_engine_name, shard_downloader)
 print(f"Using inference engine: {inference_engine.__class__.__name__} with shard downloader: {shard_downloader.__class__.__name__}")
 
-if args.node_port is None:
-  args.node_port = find_available_port(args.node_host)
-  if DEBUG >= 1: print(f"Using available port: {args.node_port}")
+if args.discovery_module == "manual":
+  if args.discovery_config is None:
+    args.discovery_module == "udp"
+  else:
+    list_device = ReadManualConfig(discovery_config=args.discovery_config)
```

Comment:
`ReadManualConfig` should be a function not a class

Reply:
Frankly, I think it is more flexible.
For exemple, for now, I set args.wait_for_peers = 1 in main.py.
But after we finish debug, I want the number of peer to wait to comme from ReadManualConfig.